db/migrate/20160615205001_add_paid_at_to_auctions.rb

```diff
@@ -0,0 +1,24 @@
+class AddPaidAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :paid_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET paid_at = CURRENT_TIMESTAMP
```

Comment:
Yeah, I think we'd have to look at the C2 records for when the payment was processed